src/functions/i18n.translator.ts

```diff
@@ -0,0 +1,11 @@
+import {
+  userLanguage,
+  labels,
+  labelWasNotFoundMesage,
+} from '../app/constants/i18n.constants';
+import { labelsKeys } from 'src/app/models/i18n.models';
+
+export const i18nTranslator = (
+  label: labelsKeys,
+  language: string = userLanguage
```

Comment:
optional argument is required since client language definition can be implemented in various ways, so I just add flex approach how we can use it with client's language via `navigator`, but it also can be different value passed as any localisation service  method like `i18nTranslator(label, this.anyLocalisationService.getClientLanguage)`;

separately labels can be formed on initialisation phase if it will be necessary.